app/assets/stylesheets/global.sass

```diff
@@ -70,11 +70,10 @@ html, body
     margin-top: 12px
     outline: none
     text-align: center
-    @media screen and (min-width: $screen-md)
+    @media screen and (min-width: $screen-sm)
       display: none
     i
       position: relative
-      top: 5px
```

Comment:
This <i> is inside a.button-burger so the styling won't get carried on to other <i>s. I'd argue to keep it as is since button-burger only shows for small screens and I already checked that the burger looks good on these screens.